packages/dotcom-ui-header/src/header.scss

```diff
@@ -67,5 +67,33 @@
 }
 
 .ft-header__drawer-ask-ft-button {
-  margin-left: 16px;
+  display: block;
+  width: 100%;
+  box-sizing: border-box;
+  margin: 0;
+  text-align: center;
+
+}
+
+.edition-switcher {
+  display: flex;
+  align-items: center;
+  font-size: 16px;
+  padding-left: 16px;
+  color: var(--Black-Tints-Black-60, #66605C);
+
+  .o-header__drawer-menu-link {
+    padding-top: 6px;
+    padding-bottom: 6px;
+    font-size: 16px;
+    font-family: MetricWeb;
```

Comment:
```suggestion

```

I think font-family is set on `.o-header-drawer` so we don't need to set it again.